Gems/ScriptCanvas/Code/Tests/Libraries/Math/ScriptCanvasUnitTest_Transform.cpp

```diff
@@ -43,6 +43,14 @@ namespace ScriptCanvasUnitTest
         EXPECT_EQ(actualResult.GetTranslation(), AZ::Vector3::CreateOne());
     }
 
+    TEST_F(ScriptCanvasUnitTestTransformFunctions, FromRotationScaleAndTranslation_Call_GetExpectedResult)
+    {
+        auto actualResult = TransformFunctions::FromRotationScaleAndTranslation(AZ::Quaternion::CreateIdentity(), 2.0, AZ::Vector3::CreateOne());
+        EXPECT_EQ(actualResult.GetUniformScale(), 2);
+        EXPECT_EQ(actualResult.GetRotation(), AZ::Quaternion::CreateIdentity());
+        EXPECT_EQ(actualResult.GetTranslation(), AZ::Vector3::CreateOne());
```

Comment:
No need to change this now, but just for future reference we have a number of `IsClose` matchers to replace multiple calls to `EXPECT_EQ` (they're usually better to use as they include a small epsilon check, often needed for results coming from calculations that aren't default values (you can override that value too if you wish)).